sbt-bundle-tester/src/main/configuration/backend/bundle.conf

```diff
@@ -0,0 +1,21 @@
+version    = "1.0.0"
+name       = "override"
+system     = "override-1.0.0"
+nrOfCpus   = 1.0
+memory     = 67108864
+diskSpace  = 5000000
+roles      = ["backend"]
+components = {
+  "override-1.0.0" = {
+    description      = "override"
+    file-system-type = "universal"
+    start-command    = ["override-1.0.0/bin/override", "-J-Xms67108864", "-J-Xmx67108864"]
```

Comment:
A nicer example of a configuration override would be to have to just:

```
components = {
  "override-1.0.0" = {
    start-command    = ["override-1.0.0/bin/override", "-J-Xms67108864", "-J-Xmx67108864"]
  }
}
```

i.e. just so what is being overloaded.